src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/HttpResourceManagerRpc.java

```diff
@@ -301,4 +301,16 @@ public Policy replacePolicy(String projectId, Policy newPolicy) throws ResourceM
       throw translate(ex);
     }
   }
+
+  @Override
+  public Policy replaceOrgPolicy(String resource, Policy newPolicy) {
+    try {
+      return resourceManager
+          .organizations()
+          .setIamPolicy(resource, new SetIamPolicyRequest().setPolicy(newPolicy))
+          .execute();
+    } catch (IOException ex) {
+      throw translate(ex);
```

Comment:
converting an IOException to a runtime exception is an antipattern

Reply:
done